components/director/internal/domain/apptemplate/resolver.go

```diff
@@ -184,36 +187,44 @@ func (r *Resolver) RegisterApplicationFromTemplate(ctx context.Context, in graph
 
 	ctx = persistence.SaveToContext(ctx, tx)
 
+	log.Infof("Registering an Application from Application Template with name %s", in.TemplateName)
```

Comment:
can we log any details about the applicaton at this point (id, name?)

Reply:
The `id` is not available as the Application is not yet created. Regarding the Application `name`, where the log is it isn't yet available. 

However, few lines down the code(line 195) when we get the `ApplicationTemplate` by its name we can access the `ApplicationInputJSON` which holds the app name, unmarshal the JSON and get the name as a string. 

I implemented a "straight-forward" solution, you can have a look.